src/auspex/filters/average.py

```diff
@@ -136,6 +136,8 @@ def update_descriptors(self):
 
         # Define variance axis descriptor
         descriptor_var = descriptor_in.copy()
+        if descriptor_var.dtype == np.complex128:
+            descriptor_var.dtype = np.float64
```

Comment:
Does this conflict with any changes in #176 ?

Reply:
That's right, reverted last commit